includes/models/model.llms.user.certificate.php

```diff
@@ -689,23 +692,28 @@ public function sync( $context = 'update' ) {
 		}
 
 		$this->set( 'title', get_post_meta( $template_id, '_llms_certificate_title', true ) );
-		set_post_thumbnail( $this->get( 'id' ), get_post_thumbnail_id( $template_id ) );
+		if ( ! set_post_thumbnail( $this->get( 'post' ), get_post_thumbnail_id( $template_id ) ) ) {
+			delete_post_thumbnail( $this->get( 'post' ) );
+		}
```

Comment:
Good catch. Please update the existing tests to also test this new scenario that I had previously missed.